specs/lib/commentOnClosedIssueSpec.js

```diff
@@ -190,12 +237,75 @@ describe('commentOnClosedIssue', function () {
         });
         spyOn(requestPromise, 'post');
 
-        commentOnClosedIssue._implementation('issueUrl', 'commentsUrl', repositorySettings)
+        var options = {
+            url: issueUrl,
+            commentsUrl: commentsUrl,
+            repositorySettings: repositorySettings,
+            outreachUsers: outreachUsers
+        };
+
+        commentOnClosedIssue._implementation(options)
             .then(done.fail)
             .catch(function (error) {
                 expect(error).toEqual('Bad request');
                 expect(requestPromise.post).not.toHaveBeenCalled();
                 done();
             });
     });
+
+    it('commentOnClosedIssue._implementation posts about first timer\'s contribution.', function (done) {
```

Comment:
Let's also test that the message don't get included when it's not their first contribution.